script/read_from_file.py

```diff
@@ -17,19 +17,34 @@ def insert_data(my_file, tag):
             try:
                 database_insert.insert_mydata(i)
             except IntegrityError:
-                ur_choice = input(
-                    "you have similar id, if u want to replace ur data type 1: "
-                )
                 database.db.session.rollback()
-                if ur_choice == "1":
-                    database_update.update(i, i["id"])
+                if not from_api:
+                    ur_choice = input(
+                        "you have similar id, if u want to replace ur data type 1: "
+                    )
+                    if ur_choice == "1":
+                        database_update.update(i, i["id"])
+                else:
+                    if force_replace:
+                        database_update.update(i, i["id"])
 
 
 def read_from_file(tag):
     f_path = f"..\my_file\data_{tag}.json"
     f = open(f_path, "r+")
     my_file = json.load(f)
-    insert_data(my_file, tag)
+    insert_data(my_file, tag, False, False)
+
+
+def read_from_file_api(my_file, tag, force_replace):
+    my_file_content = json.load(my_file)
```

Comment:
what if the file loading failed? need to handle and tell the user that the file is not in a correct format